docs-dev/NODE_METRICS.md

```diff
@@ -0,0 +1,41 @@
+# Node Metrics
+
+By default, Ceramic Nodes publish some limited metrics about their state publicly to a model on the Ceramic Network.
+
+## Metric Data Published
+
+The data published is defined in the [@ceramicnetwork/node-metrics](https://www.npmjs.com/package/@ceramicnetwork/node-metrics) package.
+
+Currently, this includes the following about the Ceramic node:
+
+  - DID used to log into the Ceramic Anchor Service
+  - IP address, if known
+  - Peer ID
+  - js-ceramic version
+  - ceramic-one version, or IPFS version if still using IPFS.
+
+As well as the following dynamic metrics published periodically (by default once per minute):
+
+  - total Pinned streams
+  - total Indexed models
+  - current number of pending requests to CAS
+  - mean pending Anchor request age
+  - max pending Anchor request age
+  - number of writes in the last window
```

Comment:
wait in the other PR you said this is measuring the number of anchor requests, but you changed it to say the number of writes, so now I'm confused.  Is this measuring the number of writes coming into this node, the number of http requests coming into this node, the number of anchor requests this node is making to the CAS, something else?  I'd like us to be precise about what is actually being measured here, both on this line and the line below

Reply:
so we actually currently measure

```
  recentCompletedRequests: Int
  ```
  
  and this is counted here - 
  https://github.com/ceramicnetwork/js-ceramic/blob/ccfb34203e6218926e213970c9d379b7680f2cea/packages/core/src/anchor/anchor-processing-loop.ts#L72
  
  We have space in the composite reserved for 'recentCreatedRequests' but we do not actually observe those at this time. If we want to do so i think taht coudl be a separate PR?
  
  I will change the documentation to mention "recently completed anchor requests"

Reply:
ok, removed this line, since the "completed anchor requests" line was already there.  When we add code to measure the number of created anchor requests then we can add it to the documentation as well.